internal/worktree/worktree_fixture_test.go

```
@@ -91,6 +94,13 @@ func gitText(t *testing.T, cwd string, args ...string) string {
 	return strings.TrimSpace(string(output))
 }
 
+func gitCommand(cwd string, args ...string) *exec.Cmd {
+	cmd := exec.Command("git", args...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)